Dockerfile

```diff
@@ -184,6 +206,7 @@ COPY --from=builder /out/velero ${BIN_DIR}
 COPY --from=builder /aws/bin/ ${BIN_DIR}
 COPY --from=builder /usr/local/aws-cli /usr/local/aws-cli
 COPY --from=builder /out/yq ${BIN_DIR}
+COPY --from=builder /out/sshuttle ${BIN_DIR}
```

Comment:
This fails for me.  Looking at the Dockerfile it appears to be installed via DNF inside the builder, so there's not an "out" to copy here.  This isn't the best way to get it into the final container image, IMO.

Remove this line, and move the "dnf install sshuttle" to the dnf-install image up above (~ line 32), and it should be in the final image.  This will require installing the epel repo to the dnf-install image/section as well.

I've tested the changes mentioned here and it's working fine.

Reply:
Ack, will change these accordingly